src/cml.js

```diff
@@ -159,19 +160,33 @@ class CML {
     throw new Error(`driver ${driver} unknown!`);
   }
 
+  watermark(driver, label = '') {
+    if (label === '') {
+      return `![](${WATERMARK_IMAGE} "CML watermark")`;
+    }
```

Comment:
minor, but do we prefer this for readability?

```suggestion
  watermark(driver, label = 'CML watermark') {
```

Reply:
That's not exactly the same, though

Reply:
https://github.com/iterative/cml/pull/1173#discussion_r990919162 for identical behavior? 😸